packages/react-components/babel-preset-global-context/package.json

```diff
@@ -14,6 +14,9 @@
     "build": "just-scripts build --commonjs",
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
+    "pree2e": "yarn build",
```

Comment:
preset needs to exist before the test starts, since this package has no dependencies within the monorepo, it should be OK to build this before testing

Reply:
can we inline this with the `e2e` command or what is the reasoning behind this? IMO this doesn't add any additional clarity rather may add confusion + the name pree2e is not very explanatory. I know npm gurus will know that `pre` is a special pragma to tell npm to execute prior to particular alias but still

Reply:
I can inline it... I tried to be an npm guru :)